bundles/pubsub/pubsub_spi/src/pubsub_utils_match.c

```diff
@@ -227,6 +228,7 @@ double pubsub_utils_matchSubscriber(
     pubsub_get_topic_properties_data_t data;
     data.isPublisher = false;
     data.topic = celix_properties_get(svcProperties, PUBSUB_SUBSCRIBER_TOPIC, NULL);
+    data.scope = celix_properties_get(svcProperties, PUBSUB_SUBSCRIBER_SCOPE, PUBSUB_SUBSCRIBER_SCOPE_DEFAULT);
```

Comment:
Just a heads up, with PR 181, the default scope is actually NULL, not "default". Only in places where a value is needed, a value is used (discovery and internal map key).

Reply:
I have prepare the code for scope = NULL, but I don't need the complete PR for testing this.